src/Api/EnumInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Api;
+
+interface EnumInterface
+{
+    public static function getValues(): array;
```

Comment:
Can we at least get rid of the `static`?

Reply:
Creating an instance of an Enum class is not a good idea.